cli/README.md

```diff
@@ -353,6 +352,10 @@ If no npm project is found, it wil run `npm init -y` in the folder where it is i
 
 ### Install @prisma/client
 
-After identifying the npm project, the generator will ensure that the `@prisma/client` package is installed. If it is already present in the `package.json` file, no action is taken. Otherwise [Joël - explain]
+After identifying the npm project, the generator will ensure that the `@prisma/client` package is installed. If it is already present in the `package.json` file, no action is taken. Otherwise it will run `npm install @prisma/client` to add the package as a dependency.
 
-> Question: How do we handle the case where `@prisma/client` package has a different version than the CLI/generator?
\ No newline at end of file
+> Question: How do we handle the case where `@prisma/client` package has a different version than the CLI/generator?
+- Currently, the CLI only prints a warning. This is useful for developing as versions are always different
+- Maybe we want
+  - to give a more useful warning and indicate how to fix it? ie (run `npm install --save-dev prisma2@2.0.0` if `"@prisma/client": "2.0.0"`)
+  - to error and require the user to fix it (even for a minor version?)
```

Comment:
@timsuchanek @sorenbs @janpio We'll need to take a decision here.

Reply:
There is an issue where this was raised by @schickling ... trying to find it.

Reply:
Found it: https://github.com/prisma/prisma2/issues/1539